Vagrantfile

```diff
@@ -207,7 +253,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if Vagrant.has_plugin?("vagrant-vbguest")
         config.vbguest.auto_update = false
     end 
-    if ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == "ubuntu" then
+    if node_os == "ubuntu" then
```

Comment:
`#{var}` is used to replace this with the value of the global variable with that same name. 

`%{var}` is used for the ruby template to do the same thing with local variables. The template is used to avoid passing arguments to the shell script (e.g.: s.args $foo, $bar, $baz become $1, $2, $3 in the shell script and we have to keep track of which variable was what when we modify the script). Instead of the numbered variables in the shell script, we have `%{foo}`, `%{bar}` `%{baz}` and we can pass a ruby hash to replace those with certain values. This is more straightforward and makes it easier to follow.

It's node_os here because we're in ruby code. The other occurrences were in strings which become templates. I've written this to explain why I've made the change.